test/export/test_export.py

```diff
@@ -1917,14 +1923,16 @@ def forward(self, x):
 graph():
     %p_p1 : [num_users=1] = placeholder[target=p_p1]
     %p_p2 : [num_users=1] = placeholder[target=p_p2]
+    %b_b1 : [num_users=1] = placeholder[target=b_b1]
     %x : [num_users=1] = placeholder[target=x]
     %mul : [num_users=1] = call_function[target=torch.ops.aten.mul.Tensor](args = (%p_p1, 2), kwargs = {})
     %add : [num_users=1] = call_function[target=torch.ops.aten.add.Tensor](args = (%mul, %p_p2), kwargs = {})
-    %sum_1 : [num_users=1] = call_function[target=torch.ops.aten.sum.default](args = (%add,), kwargs = {})
-    %getattr_33 : [num_users=1] = call_function[target=builtins.getattr](args = (%sum_1, a), kwargs = {})
-    %getattr_38 : [num_users=1] = call_function[target=builtins.getattr](args = (%getattr_33, b), kwargs = {})
```

Comment:
unrelated to this PR, but can you remind me: I thought that we decided to *only* run parametrizations implicitly in export when we are lowering to post-dispatch ATen IR, in `run_decompositions()`. Since this test is just printing the training graph (with pre-dispatch IR), why are we seeing getattrs in the graph at all?

Reply:
This is because in the forward, we actually access inner tensors so they should show up in the IR. Main limitation why we end up here is because export still doens't support tensor subclass output.